Sources/NetworkProtection/Networking/NetworkProtectionClient.swift

```diff
@@ -248,12 +256,19 @@ final class NetworkProtectionBackendClient: NetworkProtectionClient {
         do {
             let (data, response) = try await URLSession.shared.data(for: request)
             guard let response = response as? HTTPURLResponse else {
-                return .failure(.failedToFetchRegisteredServers(nil))
+                return .failure(.noResponseFromRegisterEndpoint)
```

Comment:
Changed the error to a specific one that doesn't take parameters.

This allows us to differentiate this case from the other cases below.